homeassistant/components/energy/validate.py

```diff
@@ -54,8 +58,9 @@
 WATER_USAGE_DEVICE_CLASSES = (sensor.SensorDeviceClass.WATER,)
 WATER_USAGE_UNITS = {
     sensor.SensorDeviceClass.WATER: (
-        UnitOfVolume.CUBIC_METERS,
+        UnitOfVolume.CENTUM_CUBIC_FEET,
```

Comment:
To be clear, this is the 2nd out of 2 functionality changes in the PR; the rest are just sorting?